runtime/bindings.m

```diff
@@ -88,3 +89,54 @@
 #endif
 #endif
 
+/*
+ * Vector c bindings
+ */
+
+typedef CGPoint (*vision_func) (vector_float2 faceLandmarkPoint, CGRect faceBoundingBox, size_t imageWidth, size_t imageHeight);
+
+CGPoint
+xamarin_CGPoint__VNNormalizedFaceBoundingBoxPointForLandmarkPoint_Vector2_CGRect_nuint_nuint (struct Vector2f faceLandmarkPoint, CGRect faceBoundingBox, xm_nuint_t imageWidth, xm_nuint_t imageHeight) {
+
+	static vision_func func = NULL;
+
+	if (func == NULL) {
+		void *vision_handle = dlopen ("/System/Library/Frameworks/Vision.framework/Vision", RTLD_LAZY);
+		if (vision_handle == NULL)
+			xamarin_assertion_message ("Could not open Vision Framework");
+
+		func = (vision_func) dlsym (vision_handle, "VNNormalizedFaceBoundingBoxPointForLandmarkPoint");
```

Comment:
make sense, We'll revisit that along other usage of `dlsym` at some point.
In the mean time let's add a comment this can be simplified once our minimum requirement is Xcode9.